ql/src/java/org/apache/hadoop/hive/ql/metadata/formatting/TextMetaDataFormatter.java

```diff
@@ -88,17 +89,21 @@ public void error(OutputStream out, String msg, int errorCode, String sqlState)
       }
 
   @Override
-  public void error(OutputStream out, String errorMessage, int errorCode, String sqlState, String errorDetail)
+  public void error(OutputStream out, String errorMessage, int errorCode, String sqlState, Throwable cause)
       throws HiveException
       {
     try {
-      out.write(errorMessage.getBytes("UTF-8"));
-      if(errorDetail != null) {
-        out.write(errorDetail.getBytes("UTF-8"));
+      out.write(errorMessage.getBytes(StandardCharsets.UTF_8));
```

Comment:
Duplicated code. Are we doing this, so the stringify stuff will not be used anymore in inappropriate places?